src/Sylius/Bundle/CoreBundle/Form/Type/ProductReviewType.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Form\Type;
+
+use Sylius\Bundle\ReviewBundle\Form\Type\ReviewType;
+
+/**
+ * @author Mateusz Zalewski <mateusz.p.zalewski@gmail.com>
+ */
+class  ProductReviewType extends ReviewType
```

Comment:
But ``CustomerGuestType`` shouldn't be added in this class (instead of ``ReviewType``)? Then, in ReviewBundle, we wouldn't have any information about User. 
@michalmarcinkowski WDYT?